src/soilbiogeochem/SoilBiogeochemDecompCascadeMIMICSMod.F90

```diff
@@ -0,0 +1,1024 @@
+
+         ! tau_m1_s* are tauR and tau_m2_s* are tauK in Wieder et al. 2015
+         ! and are time-dependent terms
+         tau_m1_s1 = tau_r1 * exp(tau_r2 * fmet) * tauMod
+         tau_m1_s2 = tau_r1 * exp(tau_r2 * fmet) * tauMod
+         tau_m1_s3 = tau_r1 * exp(tau_r2 * fmet) * tauMod
+         tau_m2_s1 = tau_k1 * exp(tau_k2 * fmet) * tauMod
+         tau_m2_s2 = tau_k1 * exp(tau_k2 * fmet) * tauMod
+         tau_m2_s3 = tau_k1 * exp(tau_k2 * fmet) * tauMod
+
+         do j = 1,nlevdecomp
+            ! vmax are time-dependent terms
+            ! Table B1 Wieder et al. (2015) & MIMICS params file give different
+            ! kslope. I used the params file value(s).
+            ! TODO Introduce t_soi24_degC_col following t_veg24_patch (running
+            !        mean) or t_mo_patch (30-d avg) or ...
+            vmax_l1_m1 = exp(vslope_l1_m1 * t_soi24_degC_col(c,j) + vint_l1_m1) * vmod_l1_m1
+            vmax_l1_m2 = exp(vslope_l1_m2 * t_soi24_degC_col(c,j) + vint_l1_m2) * vmod_l1_m2
+            vmax_l2_m1 = exp(vslope_l2_m1 * t_soi24_degC_col(c,j) + vint_l2_m1) * vmod_l2_m1
+            vmax_l2_m2 = exp(vslope_l2_m2 * t_soi24_degC_col(c,j) + vint_l2_m2) * vmod_l2_m2
+            vmax_s3_m1 = exp(vslope_s3_m1 * t_soi24_degC_col(c,j) + vint_s3_m1) * vmod_l2_m1
+            vmax_s3_m2 = exp(vslope_s3_m2 * t_soi24_degC_col(c,j) + vint_s3_m2) * vmod_l2_m2
+
+            ! km are time-dependent terms
+            km_l1_m1 = exp(kslope_l1_m1 * t_soi24_degC_col(c,j) + kint_l1_m1) * kmod_l1_m1
+            km_l1_m2 = exp(kslope_l1_m2 * t_soi24_degC_col(c,j) + kint_l1_m2) * kmod_l1_m2
+            km_l2_m1 = exp(kslope_l2_m1 * t_soi24_degC_col(c,j) + kint_l2_m1) * kmod_l2_m1
+            km_l2_m2 = exp(kslope_l2_m2 * t_soi24_degC_col(c,j) + kint_l2_m2) * kmod_l2_m2
+            km_s3_m1 = exp(kslope_s3_m1 * t_soi24_degC_col(c,j) + kint_s3_m1) * kmod_s3_m1 / p_scalar(c,j)
+            km_s3_m2 = exp(kslope_s3_m2 * t_soi24_degC_col(c,j) + kint_s3_m2) * kmod_s3_m2 / p_scalar(c,j)
+
+            ! Product of w_scalar * depth_scalar * o_scalar
+            w_d_o_scalars = w_scalar(c,j) * depth_scalar(c,j) * o_scalar(c,j)
+
+            ! decomp_k used in SoilBiogeochemPotentialMod.F90
+            ! TODO Defined correctly here?
+            ! TODO Combine multiple terms of same donor pools...
+            ! TODO Where to calculate m1_pool and m2_pool?
+            ! TODO When to introduce litter/soil biomass to the calculation?
+            ! TODO Excluded spinup terms for now. See cwd below how used.
+            decomp_k(c,j,i_met_lit) = (vmax_l1_m1 * m1_pool(c,j) / &
+                                       (km_l1_m1 + m1_pool(c,j))) * &
+                                      w_d_o_scalars
+            decomp_k(c,j,i_met_lit) = (vmax_l1_m2 * m2_pool(c,j) / &
+                                       (km_l1_m2 + m2_pool(c,j))) * &
+                                      w_d_o_scalars
```

Comment:
look into changing 3rd index to donor-receiver pair indices.

Reply:
confirm whether decomp_k is used in a transitions loop or a pool loop. We need transitions loop.